infrastructure/testdriver/bidi/bluetooth/handle_request_device_prompt.https.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<meta charset="utf-8"/>
+<title>TestDriver bidi.bluetooth.handle_request_device_prompt method</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js?feature=bidi"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="resources/bidi-bluetooth-helper.js"></script>
+
+<script>
+    const name = 'LE Device';
+    promise_setup(async () => {
+        await test_driver.bidi.bluetooth.simulate_adapter({
+            state: "powered-on"
+        });
+        await test_driver.bidi.bluetooth.simulate_preconnected_peripheral({
+            address: "09:09:09:09:09:09",
+            name: name,
+            manufacturerData: [],
+            knownServiceUuids: []
+        });
+        await test_driver.bidi.bluetooth.request_device_prompt_updated.subscribe();
+    });
+
+    promise_test(async (t) => {
+        test_driver.bidi.bluetooth.request_device_prompt_updated.once().then(
```

Comment:
The scenario with an empty promptEvent.devices is problematic here. Even though this assertion will add a meaningful error output "Uncaught Error ...", the test will not fail on it. The requestDeviceWithTrustedClick will hang due to unhandled prompt, and the test will timeout. I'd propose to wait for this promise along with the requestDeviceWithTrustedClick via await Promise.all([...]).

```suggestion
        const handle_prompt_promise = test_driver.bidi.bluetooth.request_device_prompt_updated.once().then(
```

Reply:
Done.